backend/src/services/ecfmp.service.ts

```diff
@@ -211,20 +211,15 @@ export async function doesMeasureExist(ident: string): Promise<boolean> {
   }
 }
 
-export async function setMeasureEnable(measureId: string, enabled: boolean) {
+export async function editMeasure(measure: EcfmpMeasureDocument) {
   try {
-    const measure: EcfmpMeasureDocument | null = await ecfmpModel.findOne({ id: measureId }).exec();
-
-    if (measure) {
-      measure.enabled = enabled;
-
-      await measure.save();
-    }
-
-    return await ecfmpModel.find().exec();
+    const updatedMeasure = await ecfmpModel.findOneAndUpdate({ id: measure.id }, { enabled: measure.enabled }, {new: true}).exec();
```

Comment:
I don't think it's good practice to take the id out of the request body. IMO that should be another parameter to the function.